File: G1_PPAI/G1_PPA1_E1/Entidades/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace G1_PPA1_E1.Entidades
{
    public class Cliente
    {
        //Atributos
        private int dni;
        private string nombreCompleto;
        private int nroCelular;

        //Constructor
        public Cliente(int dni, string nombreCompleto, int nroCelular)
        {
            this.dni = dni;
            this.nombreCompleto = nombreCompleto;
            this.nroCelular = nroCelular;
        }

        //Metodos
        public string getNombre()
        {
            return nombreCompleto;
        }

    }
}

File: PPAI_E3/src/main/java/dsi/g1/entidades/Cliente.java
package dsi.g1.entidades;

public class Cliente {
    //atributos
    private int dni;
    private String nombreCompleto;
    private int nroCelular;
    //constructor

    public Cliente(int dni, String nombreCompleto, int nroCelular) {
        this.dni = dni;
        this.nombreCompleto = nombreCompleto;
        this.nroCelular = nroCelular;
    }
    // metodos

    public String getNombre() {
        return nombreCompleto;
    }
}
